.tool-versions

```diff
@@ -1,9 +1,29 @@
+# renovate: datasource=github-releases depName=pre-commit/pre-commit
```

Comment:
tbf, I think that `.pre-commit-hooks/sort_tool_versions.py` is an overkill here. I don't see that list of tools changed often, and if it does, it should be a convention for the author to keep it sorted.

In case you want to have automation, I then would suggest a check that verifies if a list is sorted or not (and then the fix is up to the author). Something like

Reply:
> I can use the proposed bash implementation, I will try it!

I'd say let's not spend too much time on it, if it's a quick win sure, otherwise as I mentioned totally fine to have PR authors check the sorting when introducing a new tool.